src/views/assets/Domain/DomainDetail/GatewayList.vue

```diff
@@ -133,18 +133,7 @@ export default {
                     }
                   }.bind(this)
                 }
-              ],
-              onClone: function({ row }) {
-                const cloneRoute = {
-                  name: 'GatewayCreate',
-                  query: {
-                    domain: this.object.id,
-                    platform_type: row.type.value,
-                    clone_from: row.id
-                  }
-                }
-                this.$router.push(cloneRoute)
-              }.bind(this)
+              ]
             }
           }
 
```

Comment:
There may be some minor formatting discrepancies between the two files that you've shared, such as extra spaces before closing tags or inconsistent indentation styles. These won't impact the functionality of the code but could potentially make it easier to read if they're fixed. For more comprehensive improvements, we'd recommend applying a linter tool to catch these types of errors at compile time (e.g., ESLint). If there's no significant change in functionality related to the changes made here, the current structure and implementation is fine. Let me know if you need help with anything else!